pkg/i2gw/providers/ingresskong/converter.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ingresskong
+
+import (
+	"github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw"
+	"github.com/kubernetes-sigs/ingress2gateway/pkg/i2gw/providers/common"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// converter implements the ToGatewayAPI function of i2gw.ResourceConverter interface.
+type converter struct {
+	conf *i2gw.ProviderConf
+
+	featureParsers []i2gw.FeatureParser
+}
+
+// newConverter returns an ingress-kong converter instance.
+func newConverter(conf *i2gw.ProviderConf) *converter {
+	return &converter{
+		conf: conf,
+		featureParsers: []i2gw.FeatureParser{
+			headerMatchingFeature,
+			methodMatchingFeature,
+		},
+	}
+}
+
+// ToGatewayAPIResources converts the received i2gw.InputResources to i2gw.GatewayResources
+// including the ingress-kong specific features.
+func (c *converter) ToGatewayAPI(resources i2gw.InputResources) (i2gw.GatewayResources, field.ErrorList) {
```

Comment:
Can you please rename the documentation to `ToGatewayAPI`? (I know ingress-nginx implementation is also inconsistent, this function was renamed multiple times, so I missed it at the time)

Reply:
Yep, I've renamed both the kong and the nginx one.